index.html

```diff
@@ -4891,6 +4891,20 @@ <h3>Status</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Schema</h3>
+
+        <p>
+If the <code>credentialSchema</code> property is available, the schema of a
+<a>verifiable credential</a> is expected to be evaluated by the <a>verifier</a>
```

Comment:
```suggestion
<a>verifiable credential</a> can be evaluated by the <a>verifier</a>
```

Reply:
(Company representative hat on): Our software will probably not utilize `credentialSchema` in a significant number of use cases -- the reasoning being that the issuer wouldn't have issued an VC with an invalid schema, or they're doing bad schema management and we don't want `credentialSchema` mismatches to get in the way of the VC being utilized. IOW, I'm not sure that we're positive or negative on this feature yet and therefore do not want any expectations set on verifier's using this feature. It's fine if it's there, it's fine if it's optional, so the request here is to not set any expectations that the verifier is expected to use the schema property (unless they want to).

Reply:
my intention was to convey if the property was there it is expected to be evaluated. if the issuer puts it there, they intend for it to be used.

Reply:
if the issuer puts it there and wants to say it's optional - I'm not sure how they convey that. I view the properties in the VCDM as a (loose) contract between an issuer and verifier

Reply:
Nevermind, I failed to realize that the section you added is in the Validation section of the spec. :)

If the intention is to make processing the `credentialSchema` field mandatory, then we need to discuss that as a WG. 

With my company representative hat on, we would be a -1 to mandatory processing of `credentialSchema`. We believe it should be an option for the verifier to set because it is, ultimately, their decision. The feature should mirror how we do `credentialStatus`.

With my editor hat on, having language like "is expected to be", when what you really mean is "MUST", is dicey, because people will read it as a MAY instead of a MUST. Even SHOULD is dicey if you are suggesting that organizations will process the field by default.

To provide an analogy, the `credentialStatus` field processing is optional, the spec doesn't say you MUST process it. It just says what the field is used for and what the field should contain:

https://w3c.github.io/vc-data-model/#status

And then we use the  "is expected to be" language in the section on Validation... which is what this PR does (IIUC).

Reply:
thanks @msporny I'm open to a future change to keep the language more consistent with `credentialStatus`